package.json

```diff
@@ -38,7 +43,7 @@
   },
   "dependencies": {
     "@shiftcommerce/shift-node-api": "^0.1.1",
-    "@shiftcommerce/shift-react-components": "^0.1.0",
+    "@shiftcommerce/shift-react-components": "shiftcommerce/shift-react-components#feature/790_checkout_integration",
```

Comment:
This will be replaced before merge with the appropriate version, ie. that includes the `shift-react-components `required for the feature